### PR TITLE
mount /sys in getty container and add and fix kernel tests

### LINKED ISSUE #2019: mount securityfs on boot failed
**Description**

when run "mount | grep "securityfs", " it is null. So there maybe is some issues for the rc.local, or else.
**Steps to reproduce the issue:**

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you deem important (e.g. issue happens only occasionally):**

### PR DESCRIPTION
**Pull request by justincormack**
Mount /sys in the `getty` container to fix #2019

Add some tests for the filesystems we expect to find.

Fix the kernel config tests so they actually fail when they fail...

### CODE REVIEW

**Comment by justincormack** [comment]
Failures a real as I fixed the tests, there are some 4.11 module changes

**Comment by justincormack** [comment]
ok, two config options were renamed in 4.11, and one now seems to be off as presumably has preconditions, will do a different PR for these and rebase

**Comment by rn** [comment]
Ah, should have maybe caught those when doing the 4.11 port. Not sure I did a detailed diff when I updated

**Comment by justincormack** [comment]
Its actually very confusing, trying to work out exactly what did change...

**Comment by justincormack** [comment]
Ok, this has passed as I pushed the new 4.11

File: test/pkg/kernel-config/check-kernel-config.sh [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 set -e
 
+function fail {
```
Comment (riyazdf) [inline test/pkg/kernel-config/check-kernel-config.sh]: 👍  thanks for this!

File: test/pkg/kernel-config/check-kernel-config.sh [hunk 2]
```diff
@@ -19,59 +24,116 @@ kernelMinor="${kernelMinor%%.*}"
...
 fi
 
 if [ "$kernelMajor" -ge 4 -a "$kernelMinor" -ge 10 ]; then
-  echo $UNZIPPED_CONFIG | grep -q CONFIG_BUG_ON_DATA_CORRUPTION=y || (echo "CONFIG_BUG_ON_DATA_CORRUPTION=y" && exit 1)
+  echo $UNZIPPED_CONFIG | grep -q CONFIG_BUG_ON_DATA_CORRUPTION=y || fail "CONFIG_BUG_ON_DATA_CORRUPTION=y"
+fi
+
+if [ "$kernelMajor" -ge 4 -a "$kernelMinor" -le 10 ]; then
+  echo $UNZIPPED_CONFIG | grep -q CONFIG_DEBUG_RODATA=y || fail "CONFIG_DEBUG_RODATA=y"
+  echo $UNZIPPED_CONFIG | grep -q CONFIG_DEBUG_SET_MODULE_RONX=y || fail "CONFIG_DEBUG_SET_MODULE_RONX=y"
```
Comment (riyazdf) [inline test/pkg/kernel-config/check-kernel-config.sh]: 👍

**Pull request merged**